minutes/2023-06-20.md

```diff
@@ -0,0 +1,22 @@
+# Parsec Community Meeting (20/06/2023) 📋
+
+Minutes of previous meetings are stored
+[here](https://github.com/parallaxsecond/community/tree/main/minutes).
+
+## Attendance
+
+- Adam Johnston (@adamjohnstonarm)
+- Ionut Mihalcea (@ionut-arm)
+- Tomás Agustín González Orlando (@tgonzalezorlandoarm)
+
+## Agenda
+
+1.
+
+## Minutes
+
+1. Review/Overview of current active PRs.
+2. We should have the formatting check in only one CI build so that the formatting issues don't
+   crowd out the other issues and so that all CI jobs don't fail in the same way.
```

Comment:
Can you also create a GitHub issue for this? :)

Reply:
Will do :)

Reply:
https://github.com/parallaxsecond/parsec/issues/680

Reply:
Done